app/controllers/application_controller.rb

```diff
@@ -1,5 +1,17 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @current_user ||= User.first
+  # def current_user
+  #   @current_user ||= User.first
+  # end
```

Comment:
- [ ] Great job commenting out the current user method. Please kindly delete it for cleaner code practices.